### PR TITLE
Improve team section

### PR DESCRIPTION
**Pull request by GresosJ**

### CODE REVIEW

**Comment by pedrofp4444** [comment]
As long as we don't have the team pictures yet I don't think it makes sense to merge this PR, please add them 🙂🙏

File: src/data/team.json [hunk 1]
```diff
@@ -3,119 +3,120 @@
     "name": "Organização Geral",
     "members": [
       {
-        "name": "Steven Mchail",
-        "role": "Designer at Globex Corporation",
-        "image": "steven-mchail.jpg"
+        "name": "Gerson Junior",
```
Comment (pedrofp4444) [inline src/data/team.json]:
```suggestion
        "name": "Gerson Júnior",
```

**Comment by GresosJ** [comment]
I think it makes sense because I changed some crucial things, like the link I put in the images and rename "role" to "organization"

**Comment by feliciofilipe** [comment]
I think all these people (unless they are already in another team) should be apart of the tech team:
image

File: src/data/team.json [hunk 2]
```diff
@@ -3,119 +3,120 @@
...
-        "image": "waylon-hyden.jpg"
-      },
-      {
-        "name": "Giordano Sagucio",
-        "role": "Game Developer at Soylent Corp",
-        "image": "giordano-sagucio.jpg"
+        "name": "Enzo Vieira",
+        "organization": "CeSIUM",
+        "image": "default.jpg",
+        "social": "/"
```
Comment (pedrofp4444) [inline src/data/team.json]:
```suggestion
        "social": "https://github.com/EnzoVieira"
```

File: src/data/team.json [hunk 3]
```diff
@@ -3,119 +3,120 @@
...
+        "name": "Joana Branco",
+        "organization": "CeSIUM",
+        "image": "default.jpg",
+        "social": "/"
       },
       {
-        "name": "Ronni Cantadore",
-        "role": "Design Engineer at Weyland-Yutani",
-        "image": "ronni-cantadore.jpg"
+        "name": "Adriana Fraizão",
```
Comment (pedrofp4444) [inline src/data/team.json]:
```suggestion
        "name": "Adriana Frazão",
```

File: src/data/team.json [hunk 4]
```diff
@@ -3,119 +3,120 @@
...
-        "image": "erhart-cockrin.jpg"
+        "name": "Francisco Neves",
+        "organization": "CeSIUM",
+        "image": "default.jpg",
+        "social": "/"
+      }
+    ]
+  },
+  {
+    "name": "Empresas & Patrocínios",
```
Comment (pedrofp4444) [inline src/data/team.json]:
```suggestion
    "name": "Parcerias & Patrocínios",
```

File: src/data/team.json [hunk 5]
```diff
@@ -3,119 +3,120 @@
...
+        "name": "Carolina Costa",
+        "organization": "ELSA",
+        "image": "default.jpg",
+        "social": "/"
       },
       {
-        "name": "Ronni Cantadore",
-        "role": "Design Engineer at Weyland-Yutani",
-        "image": "ronni-cantadore.jpg"
+        "name": "Helder Gomes",
```
Comment (pedrofp4444) [inline src/data/team.json]:
```suggestion
        "name": "Hélder Gomes",
```

File: src/pages/index.jsx [hunk 6]
```diff
@@ -26,9 +26,9 @@ export default function Home() {
           <Speakers />
           <Schedule />
           <Papers />
+          <Team />
           <Sponsors />
           <Organization />
-          <Team />
```
Comment (pedrofp4444) [inline src/pages/index.jsx:31]: I honestly prefer the team where it was :sweat_smile:

**Review by pedrofp4444 — CHANGES_REQUESTED** [review]

File: src/pages/index.jsx (hunk 6 above)
Comment (feliciofilipe) [reply src/pages/index.jsx:31]: Me too :point_up:

Comment (GresosJ) [reply src/pages/index.jsx:31]: I don't prefer it

File: src/data/team.json [hunk 7]
```diff
@@ -3,119 +3,150 @@
...
+        "name": "Rui Lopes",
+        "organization": "CeSIUM",
+        "image": "default.jpg",
+        "social": "/"
       },
       {
-        "name": "Cathlene Burrage",
-        "role": "Frontend Developer at Buy n Large",
-        "image": "cathlene-burrage.jpg"
+        "name": "Felipe Felicio",
```
Comment (feliciofilipe) [inline src/data/team.json]:
```suggestion
        "name": "Filipe Felício",
```

File: src/data/team.json [hunk 8]
```diff
@@ -3,119 +3,150 @@
...
+        "organization": "CeSIUM",
+        "image": "default.jpg",
+        "social": "/"
```
Comment (pedrofp4444) [inline src/data/team.json]:
```suggestion
        "social": "https://github.com/feliciofilipe"
```

File: src/data/team.json [hunk 9]
```diff
@@ -3,119 +3,150 @@
...
       },
       {
-        "name": "Waylon Hyden",
-        "role": "DevOps at RDA Corporation",
-        "image": "waylon-hyden.jpg"
+        "name": "Rui Oliveira",
+        "organization": "CeSIUM",
+        "image": "default.jpg",
+        "social": "/"
```
Comment (pedrofp4444) [inline src/data/team.json]:
```suggestion
        "social": "https://github.com/ruioliveira02"
```

File: src/data/team.json [hunk 10]
```diff
@@ -3,119 +3,150 @@
...
       },
       {
-        "name": "Rinaldo Beynon",
-        "role": "Data Scientist at Rekall",
-        "image": "rinaldo-beynon.jpg"
+        "name": "Mario Rodrigues",
+        "organization": "CeSIUM",
+        "image": "default.jpg",
+        "social": "/"
```
Comment (pedrofp4444) [inline src/data/team.json]:
```suggestion
        "social": "https://github.com/marioRodrigues10"
```

File: src/data/team.json [hunk 11]
```diff
@@ -3,119 +3,150 @@
...
       },
       {
-        "name": "Giordano Sagucio",
-        "role": "Game Developer at Soylent Corp",
-        "image": "giordano-sagucio.jpg"
+        "name": "Tiago Bacelar",
+        "organization": "CeSIUM",
+        "image": "default.jpg",
+        "social": "/"
```
Comment (pedrofp4444) [inline src/data/team.json]:
```suggestion
        "social": "https://github.com/tiago-bacelar"
```

File: src/data/team.json [hunk 12]
```diff
@@ -3,119 +3,150 @@
...
       },
       {
-        "name": "Ibrahim Frasch",
-        "role": "Programmer at Umbrella Corp",
-        "image": "ibrahim-frasch.jpg"
+        "name": "Rui Lopes",
+        "organization": "CeSIUM",
+        "image": "default.jpg",
+        "social": "/"
```
Comment (pedrofp4444) [inline src/data/team.json]:
```suggestion
        "social": "https://github.com/RuiL1904"
```

**Review by pedrofp4444 — CHANGES_REQUESTED** [review]

File: src/data/team.json [hunk 13]
```diff
@@ -3,119 +3,150 @@
     "name": "Organização Geral",
     "members": [
       {
-        "name": "Steven Mchail",
-        "role": "Designer at Globex Corporation",
-        "image": "steven-mchail.jpg"
+        "name": "Gerson Júnior",
+        "organization": "CeSIUM",
+        "image": "gersonjunior.jpg",
```
Comment (pedrofp4444) [inline src/data/team.json]:
```suggestion
        "image": "gersonjunior.png",
```

**Review by pedrofp4444 — CHANGES_REQUESTED** [review]

**Review by pedrofp4444 — DISMISSED** [review]
Good job :rocket:

**Comment by feliciofilipe** [comment]
@GresosJ Order the team members by alphabetical order within each team collection plz :pray:

**Review by pedrofp4444 — APPROVED** [review]

**Comment by MarioRodrigues10** [comment]
@GresosJ , why there's a cat as default image? What's the point of it?

**Comment by GresosJ** [comment]
> @GresosJ , why there's a cat as default image? What's the point of it?

Because I thought it was a pretty cat , and that's why. In the future I will change for something better

**Pull request merged**